lms/static/sass/multicourse/_courses.scss

```diff
@@ -125,8 +157,215 @@
     color: $white;
     text-shadow: none;
 
+    //STATE: hover, focus
     &:hover, &:focus {
       background: $m-blue-l1;
     }
   }
+
+  .filters {
+    @include clearfix();
+    margin-top: ($baseline/2);
+    border-top: 1px solid $courseware-button-border-color;
+    border-bottom: 1px solid $courseware-button-border-color;
+    width: 100%;
+    height: auto;
+    max-height: ($baseline*10);
+
+    ul {
+      @include padding-left(0);
+      margin: 0;
+      list-style: outside none none;
+    }
+
+    li {
+      @include float(left);
+      @include margin(($baseline/2), $baseline, ($baseline/2), 0);
+      position: relative;
+      padding: ($baseline/2) ($baseline*0.75);
+      background: $courseware-button-border-color;
+      width: auto;
+
+      a {
+        @extend %t-strong;
+        color: $gray-d2;
+        text-decoration: none;
+
+        i {
+          color: $gray-l2;
+        }
+      }
+    }
+
+    .clear-filters {
+      @include line-height(29.73);
+      @extend %t-icon5;
+      @extend %t-strong;
+      margin: ($baseline/2) 0;
+      width: auto;
+      text-align: center;
+      color: $m-blue-d1;
+    }
+
+    .flt-right {
+      @include float(right);
+    }
+  }
+
+
+
+  .search-facets{
+    /* rtl and ltr mixins are included to switch default layer direction of this block */
+    @include rtl() { $layout-direction: "LTR"; }
+    @include ltr() { $layout-direction: "RTL"; }
+    @include box-sizing(border-box);
+    @extend %ui-depth1;
+    position: relative;
+    box-shadow: 1px 2px 5px $black-t0;
+    border-top: 1px solid $black;
+    border-bottom: 2px solid $black;
+    background-color: $white;
+    max-height: ($baseline*100);
+
+    @include fill-parent();
+    margin: ($baseline*2) 0 ($baseline*3.5) 0;
+
+    @include media($bp-medium) {
+      @include span-columns(2);
+      @include omega();
+    }
+
+    @include media($bp-large) {
+      @include span-columns(3);
+      @include omega();
+    }
+
+    @include media($bp-huge) {
+      @include span-columns(3);
+      @include omega();
+    }
```

Comment:
Should `@include omega();` be extracted outside the media boundaries, since it seems to stay the same no matter which breakpoint is active?